api/v1beta1/ibmpowervsmachine_types.go

```diff
@@ -91,6 +91,11 @@ type IBMPowerVSResourceReference struct {
 	// +kubebuilder:validation:MinLength=1
 	// +optional
 	Name *string `json:"name,omitempty"`
+
+	// Regular expression to match resource
```

Comment:
@Karthik-K-N I'm also wondering if we can add same comment to this place as well so that user can see that information in the CRD itself while creating the CR, what do you think?

Reply:
Yes, That will be very helpful, I will add it